packages/manager/modules/pci/src/projects/project/storages/cold-archive/users/users.html

```diff
@@ -35,12 +35,13 @@
     </oui-datagrid-column>
     <oui-datagrid-column
         data-title="'pci_projects_project_storages_containers_users_creation_date' | translate"
-        data-property="creationDate"
-        data-type="string"
+        data-property="gridPropertyCreationDate"
+        data-type="date"
         data-searchable
         data-sortable
         data-filterable
     >
+        <span data-ng-bind="$row.gridDisplayCreationDate"></span>
```

Comment:
**nitpick:** you can directly use the angular.js filters instead of maintaining this data on the array.

Reply:
Done.